packages/loadingIndicator/components/SectionLoadingIndicator.tsx

```diff
@@ -1,13 +1,23 @@
+import { css } from "@emotion/css";
 import * as React from "react";
 import { sectionLoadingWrapper, sectionLoadingIndicator } from "../style";
 
-const SectionLoadingIndicator = () => (
-  <div className={sectionLoadingWrapper}>
-    <div
-      className={sectionLoadingIndicator}
-      data-cy="sectionLoadingIndicator"
-    />
-  </div>
-);
+export interface SectionLoadingIndicatorProps {
+  /** Custom color for the loading indicator */
+  color?: string;
+}
+
+const SectionLoadingIndicator = ({ color }: SectionLoadingIndicatorProps) => {
+  return (
+    <div className={sectionLoadingWrapper}>
+      <div
+        data-cy="sectionLoadingIndicator"
+        className={css(sectionLoadingIndicator, {
+          backgroundColor: color
```

Comment:
I think we should default this to whatever it was before, or perhaps only set this if color is defined so it works exactly as before if not sent

Reply:
It should still default to the same color, it will only override if a color prop is passed in. The snapshots haven't changed.

Reply:
The style in `sectionLoadingIndicator` still handles the background color as previously, and it will override this with the custom background color if it's passed in. I could pass in a fallback as an extra measure if we'd like.